docs/dev/ios/setup-ios-build-environment.md

```diff
@@ -0,0 +1,41 @@
+# Setup the iOS Build Environment
+
+## Prepare your build environment
+
+1. Install Xcode 11.0
+2. Install [Carthage](https://github.com/Carthage/Carthage): `brew install carthage`
+3. Ensure you have Python 3 installed: `brew install python`
```

Comment:
Maybe add a `brew install swiftlint swiftformat`, since these are required to meet the linting requirements?